### PR TITLE
"Powered By" page for the MLflow website

### PR DESCRIPTION
**Pull request by daniellok-db**
Add a page on the new website to put all the company logos:

https://github.com/mlflow/mlflow-website/assets/148037680/702efaeb-6eb2-408a-a6a3-68687e3c3c66

I'm not super sure what the content should be, let me know if anyone has any better suggestions! I tried to include all of the companies from the old website's logo section, but some of them are now defunct / got acquired / don't have SVG logos. My thought was that if any organization wants their logo on the new site, they can reach out to us and we can ask them to provide an SVG we can use.

The logos are also sorted alphabetically, but let me know if there's any specific order we want!

### CODE REVIEW

File: website/src/data/companies.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "ABN AMRO": "https://www.abnamro.com",
```
Comment (daniellok-db) [inline website/src/data/companies.json:2]: the structure of this file can be modified. for now i kept it simple with `{ \<name>: \<url> }, but we could also do something like:

```
{
  <name>: {
    url: <url>,
    src: <path/to/logo.svg>,
    // other fields?
  },
  ...
}
```

File: website/src/pages/powered-by.tsx
Comment (harupy) [inline website/src/pages/powered-by.tsx:1]: Can we create a `powered-by` directory and changed this file to `website/src/pages/powered-by/index.tsx`?

**Comment by BenWilson2** [comment]
cc @mahesh-venkatachalam What do you think of the layout and feel of the added page?

File: website/src/pages/powered-by.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      >
+        <div className={styles.text}>
+          MLflow is trusted by organizations of all sizes to power their AI and
+          machine learning workflows. Below are some of the organizations using
+          and contributing to MLflow. To add your organization here, email our
+          user list at{" "}
+          <a href="https://groups.google.com/g/mlflow-users">
+            mlflow-users@googlegroups.com
+          </a>
+          !
```
Comment (harupy) [inline website/src/pages/powered-by.tsx:56]: Can we replace "email our user list" with "open a PR"?

Comment (harupy) [reply website/src/pages/powered-by.tsx:56]: We can file another PR to add a new logo once this PR is merged, and put that PR as an example like below:

```
open a PR at https://github.com/mlflow/mlflow-website (example)
                                                      ^^^^^^^^^
```

File: website/src/data/companies.json (hunk 1 above)
Comment (harupy) [reply website/src/data/companies.json:3]: I prefer the `name: {url, src}` style.

File: website/src/pages/powered-by.tsx (hunk 2 above)
Comment (daniellok-db) [reply website/src/pages/powered-by.tsx:56]: oh yeah that would be much better haha

**Comment by harupy** [comment]
The logo are colored when the screen is narrow. Is this intended?

image

File: website/src/pages/powered-by.tsx (hunk 2 above)
Comment (daniellok-db) [reply website/src/pages/powered-by.tsx]: removed MosaicML from this list, and will add it back in a PR later after this lands so we can provide an example

**Comment by daniellok-db** [comment]
> The logo are colored when the screen is narrow. Is this intended?
> 
> image

Yup, I did some testing and realized that the hover action doesn't work on mobile, so I just decided to make them colored

**Review by harupy — APPROVED** [review]
LGTM!

**Comment by daniellok-db** [comment]
probably this isn't going to be shipped, so closing the PR to clean up the repo

**Pull request closed without merging**